js/libs/keycloak-js/pom.xml

```diff
@@ -71,6 +71,50 @@
                     <installDirectory>../..</installDirectory>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>ca.szc.maven</groupId>
+                <artifactId>jsonpath-maven-plugin</artifactId>
+                <version>1.0.0</version>
+                <executions>
+                    <execution>
+                        <id>update-version</id>
+                        <phase>process-resources</phase>
+                        <goals>
+                            <goal>modify</goal>
+                        </goals>
+                        <configuration>
+                            <file>${basedir}/package.json</file>
+                            <modifications>
+                                <modification>
+                                    <expression>$.version</expression>
+                                    <value>${project.version}</value>
+                                </modification>
+                            </modifications>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
```

Comment:
This is not a reliable way to replace the versions in the `package.json`. We should really be using `set-version.sh` script to update the versions before creating a tag that is built.

This also takes care of setting versions in Maven files, containers, docs, and makes sure admin client/console uses the correct versions.

Reply:
Yes, this should be re-used if possible. Also because I believe the lockfile needs an update as well.